R/browse.R

```diff
@@ -59,6 +59,14 @@ browse_travis <- function(package = NULL, ext = c("org", "com")) {
   view_url(sub("github.com", travis_url, gh))
 }
 
+#' @export
+#' @rdname browse-this
+browse_circle <- function(package = NULL) {
+  gh <- github_home(package)
+  circle_url <- glue::glue("circleci.com/gh")
```

Comment:
I don't think glue is needed here?

Reply:
Would this make more sense?
```suggestion
  circle_url <- glue::glue("circleci.com/{gh}")
```

Reply:
I don't think so — we're currently using `github_home()` as a round about way to get the github repo; I'll fix that problem after this PR is merged.

Reply:
Ah, 10-4. My bad

Reply:
Nothing is glued here so yes, this can just be normal string.